src/main/java/org/folio/service/orders/lines/update/instance/WithHoldingOrderLineUpdateInstanceStrategy.java

```diff
@@ -85,7 +86,7 @@ private CompletableFuture<Void> findOrCreateHoldingsAndUpdateItems(OrderLineUpda
               .thenCompose(newHoldingId -> {
                 holder.addHoldingRefsToStoragePatchOrderLineRequest(holdingId, newHoldingId);
                 CompositePoLine compositePoLine = PoLineCommonUtil.convertToCompositePoLine(holder.getStoragePoLine());
-                if (PoLineCommonUtil.isItemsUpdateRequired(compositePoLine)) {
+                if (ObjectUtils.notEqual(holdingId, newHoldingId)) {
```

Comment:
PoLineCommonUtil.isItemsUpdateRequired(compositePoLine) - this method used in the current class twice, do we need to replace both usages?

Reply:
Hi @SerhiiNosko. I can not give definitely answer to your question due my poor knowledge the domain. Better I'll wait @grabsefx and @AndreiMPV.

Reply:
@AndreiMPV I decide to remove any conditions for "create holdings" case. No make sense checking equality holding Ids when we have holdings operation with value "CREATE"

Reply:
It makes sense, for new holding  holdingId will always will be different, so this check not needed